packages/strapi-plugin-content-type-builder/admin/src/containers/App/index.js

```diff
@@ -133,22 +133,9 @@ export class App extends React.Component {
       return newGroup;
     }
 
-    return get(
-      modifiedDataGroup,
-      [this.getFeatureNameFromSearch(), 'schema'],
-      {}
-    );
+    return get(modifiedDataGroup, this.getFeatureNameFromSearch(), {}, {});
```

Comment:
`return get(modifiedDataGroup, this.getFeatureNameFromSearch(), {});` the third argument is not needed here. https://lodash.com/docs/4.17.15#get